crates/storage/src/error.rs

```diff
@@ -9,7 +9,7 @@ pub enum StoreError {
     DecodeError,
     #[cfg(feature = "libmdbx")]
     #[error("Libmdbx error: {0}")]
-    LibmdbxError(anyhow::Error),
+    LibmdbxError(#[from] anyhow::Error),
```

Comment:
I didn't add this automatic conversion because there is a risk that some other non-libmdbx error gets converted into a `LibmdbxError` (for example, rlp module returns `anyhow::Error`s too)
Not sure how to address this issue properly, though

Reply:
Sounds reasonable, Ill remove the #[from]